main.tf

```diff
@@ -136,6 +140,7 @@ module "bastion" {
   public_subnet_id = local.public_subnets[0].subnet_id
   ami_id           = var.bastion.ami
   instance_type    = var.bastion.instance_type
+  kms_key          = var.use_kms ? data.aws_kms_key.key[0].arn : null
```

Comment:
why isn't this used above?

Reply:
you got hoodwinked by the folding, this was for the bastion. moved it to a local.